src/monopoly/board/monopoly_board.js

```diff
@@ -0,0 +1,43 @@
+var TableTop = require('../../../tabletop/tabletop');
+var _ = require('lodash');
+var inherits = require('util').inherits;
+
+// more of a placeholder for now -
+// should talk when merged to properly integrate this with
+// existing "board" and "players" implementations if any exist
+function MonopolyBoard() {
+    TableTop.Board.call(this);
+    this.tiles = [];
+}
+
+inherits(MonopolyBoard, TableTop.Board);
+
+MonopolyBoard.prototype.findTileForToken = function(token) {
+    var tile = _.find(this.tiles, function (n) {
+        return (n.tokens.indexOf(token) != -1);
+    });
+
+    return tile;
+};
+
+MonopolyBoard.prototype.moveTokenToTile = function(token, tile) {
+    var oldTile = this.findTileForToken(token);
+    if (oldTile != undefined) {
+        oldTile.tokens.splice(this.tiles.indexOf(token), 1);
+    }
+
+    tile.addToken(token);
+};
+
+MonopolyBoard.prototype.destroyToken = function(token) {
```

Comment:
Why doesn't this work for all board types? Aren't you just finding the index of the token in the tile and then removing it and then adding it to the other tile?